Source/FAIcon.swift

```diff
@@ -409,7 +409,7 @@ private class FontLoader {
             
             var error: Unmanaged<CFError>?
 
-            if CTFontManagerRegisterGraphicsFont(font, &error) == false {
+            if CTFontManagerRegisterGraphicsFont(font!, &error) == false {
```

Comment:
it might be safer to do `if let font = font, CTFontMan...` here. The `if` is already there, so binding the optional makes sense.